lnbits/core/templates/core/wallet.html

```diff
@@ -62,6 +71,15 @@ <h3 class="q-my-none">
                 style="opacity: 0.75"
               ></span>
             </div>
+            <q-btn
+              @Click="mobileSimple = !mobileSimple"
```

Comment:
```suggestion
              @click="mobileSimple = !mobileSimple"
```

Reply:
addressed in 3de064d6ca90d6c378912ede841f89b471445e6a